handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1544,15 +1547,19 @@ public final void setEnabledCipherSuites(String[] cipherSuites) {
         }
         synchronized (this) {
             if (!isDestroyed()) {
-                // TODO: Should we also adjust the protocols based on if there are any ciphers left that can be used
-                //       for TLSv1.3 or for previor SSL/TLS versions ?
                 try {
                     // Set non TLSv1.3 ciphers.
                     SSL.setCipherSuites(ssl, cipherSuiteSpec, false);
```

Comment:
Will this suffer from the same problem, namely that if we only support a single TLSv1.3 suite we'll accidentally enable all older suites as well?

Reply:
Let me verify but I think it should work